Assets/Scripts/Possession/Throwable.cs

```diff
@@ -10,6 +8,9 @@ public class Throwable : MonoBehaviour, IPossessable
     [SerializeField] [Tooltip("Extra sensitivity on y-axis for easier throwing")] private float ySense = 2;
     [SerializeField] private float _rotationSpeed = 10;
     [SerializeField] [Tooltip("Minimum Impulse needed to destroy the object")] private float _destroyMinimumImpulse = 1;
+    [SerializeField]
+    [Tooltip("The maximum speed the object is allowed to go before being throwable again")]
+    private float _maxThrowSpeed = 5f;

```

Comment:
the name _maxThrowSpeed sound like it's the maximum strength to throw an object. Please rename it to something like _notThrowableThreshold.